lib/db/mysql/index.js

```diff
@@ -153,7 +153,7 @@ MysqlStore.prototype = {
   addAvatar: function addAvatar(id, uid, url, provider, selected) {
     id = buf(id);
     uid = buf(uid);
-    var store = this;
+    var store = this; // eslint-disable-line consistent-this
```

Comment:
Well this got merged, but I think consistent-this should be removed from the base package. Forcing it to always be called self is stupid.